docs/user_guide.md

```diff
@@ -1,80 +1,8 @@
-## Getting started with the Courtgrabber CLI
+# User Guide
 
-The following user guide walks you through how to use Courtgrabber from the command line. If you would like to learn more about the Courtgrabber REST API, please refer to its [reference documentation](./api_reference.md). 
+The following user guide walks you through how to use Courtgrabber from the command line. If you would like to learn more about the Courtgrabber REST API, please refer to the [developer guide](./api_reference.md). 
 
-The Courtgrabber CLI requires minimal set-up and uses only a few simple commands. First, let's define a few key terms.
```

Comment:
I like that you moved the definitions to the Getting Started page.  I think that's the right place for them as Getting Started is likely where most users would begin their journey through the docs.  However, if someone were to start their journey - accidentally or deliberately - on the User Guide page, maybe we should make them aware of the Getting Started page and link to it?  Or do you think they should just browse their way through the docs and discover the Getting Started page instead?

Reply:
I think it's better to be explicit in this case. I've added a sentence to the intro. Thanks.